CHANGELOG.md

```diff
@@ -10,18 +10,60 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Support for 'object'-type properties.
+- Support for multi-line string properties.
+- Layer id parsing.
 
-## [0.9.2] - 2020-Apr-25
+### Changed
+
```

Comment:
I'd prefer if these spaces before the bullet lists are removed like in https://keepachangelog.com/en/1.0.0/, so that we can fit more data in less visual space.